apps/src/code-studio/components/SmallFooter.jsx

```diff
@@ -172,6 +174,17 @@ export default class SmallFooter extends React.Component {
         maxWidth: '50%',
         minWidth: this.state.baseWidth,
       },
+      copyrightXClose: {
+        position: 'absolute',
+        top: 0,
+        right: 0,
+        padding: 0,
+        color: color.charcoal,
```

Comment:
Not blocking: can this use the same `color.neutral_dark30` variable you used [here](https://github.com/code-dot-org/code-dot-org/pull/55365/files#diff-8895ca710d673a6121a62c87c0a67db76135a91ab6ff7c1541d1990abf049eb0)?